packages/blockchain-wallet-v4-frontend/src/data/components/identityVerification/sagas.ts

```diff
@@ -202,7 +186,7 @@ export default ({ api, coreSagas, networks }) => {
 
   const goToNextStep = function * () {
     const steps = (yield select(S.getSteps)).getOrElse([])
-    const currentStep = yield select(S.getVerificationStep)
+    const currentStep = S.getVerificationStep(yield select())
```

Comment:
A bit weird to me. Doesn't `getVerificationStep` take a state? if so `yield select(S.getVerificationStep)` should work

Reply:
yeah it could be like that, as I mentioned below we do not have certain consensus, but I like to use 
`yield select(smth)`more we should refactor ;)